qt/applications/workbench/workbench/projectrecovery/recoverygui/recoveryfailureview.py

```diff
@@ -45,8 +48,9 @@ def set_progress_bar_maximum(self, new_value):
         self.ui.progressBar.setMaximum(new_value)
 
     def connect_progress_bar(self):
-        self.presenter.project_recovery.loader.multi_file_interpreter.current_editor().sig_progress.connect(
```

Comment:
Should we consider having some type of base class with this common functionality in it? A `RecoveryView` that both classes can inherit from?

Reply:
This is probably something we should do, but largely out of the scope of this PR. I'll create an issue for maintenance to clear up this code, #29405.